documents/tests/test_api_list_documents.py

```diff
@@ -83,12 +90,27 @@ def test_list_document_owner(api_client, user):
     results = body.get("results", [])
 
     # The user should only be able to see the one document from the service 1
-    assert body.get("count") == 1
+    # assert body.get("count") == 1
```

Comment:
I think this doesn't need to be commented out

Reply:
Ahhh good catch 😅 I forgot it as part of my tests